static/scripts/rewards/web3/connect-wallet.ts

```diff
@@ -20,6 +20,11 @@ export async function connectWallet(): Promise<JsonRpcSigner | null> {
 
     return signer;
   } catch (error: unknown) {
+    // For testing purposes
```

Comment:
If that is meant only for testing we might wanna add some check such as `process.env.NODE_ENV === "test"`. I believe there is no `process.env` available but you can forward declare that at compile time.

Reply:
went with checking the url because the frontend url is an env var which is set already and is different for test and prod, which removes the need for loading another env var